manager/state/raft/raft.go

```diff
@@ -992,10 +994,15 @@ func (n *Node) updateNodeBlocking(ctx context.Context, id uint64, addr string) e
 	if m == nil {
 		return errors.Errorf("member %x is not found for update", id)
 	}
+	if m.BackupAddrs == nil {
+		m.BackupAddrs = make(map[string]*types.Timestamp)
+	}
+	m.BackupAddrs[m.Addr] = ptypes.MustTimestampProto(time.Now())
```

Comment:
I don't see where we delete `addr` from the map, and based on the name `BackupAddrs` and the other code it seems like we should (but see above - I think it might be better for the map to contain all addresses).